mongo/mongo.go

```diff
@@ -0,0 +1,59 @@
+package mongo
+
+import (
+	"context"
+	"errors"
+
+	"github.com/ONSdigital/dp-healthcheck/healthcheck"
+	dpMongodb "github.com/ONSdigital/dp-mongodb"
+	dpMongoHealth "github.com/ONSdigital/dp-mongodb/health"
+	"github.com/globalsign/mgo"
+)
+
+//Mongo represents a simplistic MongoDB configuration, with session and health client
+type Mongo struct {
+	Session      *mgo.Session
+	URI          string
+	Database     string
+	Collection   string
```

Comment:
`URI`, `Database`, and `Collection` are only used by the init function. Consider passing them as parameters to the init function instead of having to have them as values on the Mongo struct.

(I appreciate this may be the way it's done in other projects, if so then it's not a big deal to leave it)